.github/workflows/build.yml

```diff
@@ -4,9 +4,18 @@ on:
   push:
   pull_request:
 
+env:
+  CMAKEFLAGS_COMMON: -DCMAKE_BUILD_TYPE=Release -DBATTERY=ON -DBROADCAST=ON -DBULK=ON -DDEBUG_ASSERTIONS_FATAL=ON -DHID=ON -DLILV=ON -DOPUS=ON -DQTKEYCHAIN=ON -DVINYLCONTROL=ON
+  GTEST_COLOR: 1
+  CTEST_OUTPUT_ON_FAILURE: 1
+  # Render analyzer waveform tests to an offscreen buffer
+  QT_QPA_PLATFORM: offscreen
+  # GitHub Actions automatically zstd compresses caches
+  CCACHE_NOCOMPRESS: true
```

Comment:
Is this a time-memory tradeoff? Does disabling compression really make a big difference in build time?